docs/source/verdi/verdi_user_guide.rst

```diff
@@ -367,6 +367,7 @@ Below is a list with all available subcommands.
       structure   Manipulation of StructureData objects.
       trajectory  View and manipulate TrajectoryData instances.
       upf         Manipulation of the upf families.
+      diff        Command line interface for aiida-diff
```

Comment:
this is the danger of the automatic docs creation.
is there any way of avoiding this? conrad and me also already ran into this

Reply:
Good catch. It sure is annoying, but still the amount of things we would have missed without auto doc generation would have been far greater. I will see if I can exclude external plugins from this. Should be possible

Reply:
> the amount of things we would have missed without auto doc generation would have been far greater

fully agree

Reply:
Quite an ad-hoc solution, but still acceptable I would say in PR #2878